wasmplugin/plugin.go

```diff
@@ -110,6 +112,30 @@ func (ctx *httpContext) OnHttpRequestHeaders(numHeaders int, endOfStream bool) t
 	// and its request properties, but they may not be true of other proxies implementing
 	// proxy-wasm.
 
+	if tx.IsRuleEngineOff() {
+		return types.ActionContinue
+	}
+
+	srcAddressRaw, err := proxywasm.GetProperty([]string{"source", "address"})
```

Comment:
Looks like port is provided in separate attributes, so maybe don't need to parse

https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/advanced/attributes#connection-attributes

Reply:
I don't know why, but `port` attributes are not populated at this point. We experienced the same behaviour also  with the C++/ModSec implementation

Reply:
I'd add a TODO so this PR does not get blocked.

Reply:
I made it. PTAL